sdk/paloaltonetworks/azure-resourcemanager-paloaltonetworks-ngfw/src/main/java/com/azure/resourcemanager/paloaltonetworks/ngfw/NgfwManager.java

```diff
@@ -0,0 +1,497 @@
+import com.azure.core.util.logging.ClientLogger;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.fluent.PaloAltoNetworksCloudngfw;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.CertificateObjectGlobalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.CertificateObjectLocalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.FirewallStatusImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.FirewallsImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.FqdnListGlobalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.FqdnListLocalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.GlobalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.LocalRulesImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.LocalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.OperationsImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.PaloAltoNetworksCloudngfwBuilder;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.PostRulesImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.PreRulesImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.PrefixListGlobalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.implementation.PrefixListLocalRulestacksImpl;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.CertificateObjectGlobalRulestacks;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.CertificateObjectLocalRulestacks;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.FirewallStatus;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.Firewalls;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.FqdnListGlobalRulestacks;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.FqdnListLocalRulestacks;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.GlobalRulestacks;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.LocalRules;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.LocalRulestacks;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.Operations;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.PostRules;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.PreRules;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.PrefixListGlobalRulestacks;
+import com.azure.resourcemanager.paloaltonetworks.ngfw.models.PrefixListLocalRulestacks;
+import java.time.Duration;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/** Entry point to NgfwManager. */
+public final class NgfwManager {
```

Comment:
Maybe `PaloaltoNetworksNgfwManager`?

Reply:
Hi service team, we will need your suggestion on the `Manager` name.

Reply:
`PaloAltoNetworksNgfwManager` sounds good, as it in line with the finalized namespace.